tracee-rules/main.go

```diff
@@ -16,6 +17,9 @@ func main() {
 		Name:  "tracee-rules",
 		Usage: "A rule engine for Runtime Security",
 		Action: func(c *cli.Context) error {
+			if c.NArg() == 0 && c.NumFlags() == 0 {
```

Comment:
for my understanding, what are both needed?

Reply:
They're actually not, this stemmed from me confusing this CLI library with the other popular Go one (Cobra) which does positional argument validation. I'm removing the `NArg()` check. Tracee-rules doesn't take any positional args anyway.